File: app/src/main/java/com/zzw/customview/MainActivity.java
package com.zzw.customview;

import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.view.View;

/**
 * Created by zzw on 2017/6/15.
 * Version:
 * Des:
 */

public class MainActivity extends AppCompatActivity implements View.OnClickListener {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        findViewById(R.id.qq).setOnClickListener(this);
        findViewById(R.id.load58).setOnClickListener(this);
        findViewById(R.id.suoyin).setOnClickListener(this);
        findViewById(R.id.ziticolor).setOnClickListener(this);
        findViewById(R.id.colortv_vp).setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {
        switch (v.getId()) {
            case R.id.qq:
                startActivity(QQSportStepViewActivity.class);
                break;
            case R.id.load58:
                startActivity(JumpLoadViewActivity.class);
                break;
            case R.id.suoyin:
                startActivity(AlphabeticalIndexViewActivity.class);
                break;
            case R.id.ziticolor:
                startActivity(ColorTrackTextViewActivity.class);
                break;
            case R.id.colortv_vp:
                startActivity(ViewPagerActivity.class);
                break;
        }
    }

    private void startActivity(Class<?> clazz) {
        Intent intent = new Intent(this, clazz);
        startActivity(intent);
    }
}

File: app/src/main/java/com/zzw/customview/MainActivity.kt
package com.zzw.customview

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.text.method.Touch
import android.view.View


import kotlinx.android.synthetic.main.activity_main.*


/**
 * Created by zzw on 2017/6/15.
 * Version:
 * Des:
 */

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        qq.startActivity(QQSportStepViewActivity::class.java)
        load58.startActivity(JumpLoadViewActivity::class.java)
        suoyin.startActivity(AlphabeticalIndexViewActivity::class.java)
        ziticolor.startActivity(ColorTrackTextViewActivity::class.java)
        colortv_vp.startActivity(ViewPagerActivity::class.java)
        rating.startActivity(RatingBarActivity::class.java)
        tagLayout.startActivity(TagLayoutActivity::class.java)
        touch.startActivity(TouchActivity::class.java)
        skidding.startActivity(SkiddingActivity::class.java)
        radar.startActivity(RadarViewActivity::class.java)
        qqSkidding.startActivity(QQSliddingActivity::class.java)
    }


    fun View.startActivity(clazz: Class<*>) {
        setOnClickListener {
            val intent = Intent(context, clazz)
            startActivity(intent)
        }
    }


}
